edenia/eden-spend-explorer
# fix(webapp): trasury loading

### Trasury loading

### What does this PR do?

- Resolve #222 

### Steps to test

1. run the project and open localhost:3000 in your browser
2. go to income view a see the new treasury feature

#### CheckList
